test/batch.js

```diff
@@ -25,6 +13,11 @@ test('create 3 nodes and a way', function (t) {
   ]
   osm.batch(rows, function (err, nodes) {
     t.error(err)
+
+    osm.ready(() => { check(nodes) })
```

Comment:
Whoops. I was hacking this in quickly in the field data collection app, then copied the changes over to here, and must've missed some. (I use arrow functions sometimes to save keystrokes -- ssh!)